ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoModule.java

```diff
@@ -9,14 +9,15 @@
 
 package com.facebook.react.modules.systeminfo;
 
-import javax.annotation.Nullable;
+import android.os.Build;
+
+import com.facebook.react.bridge.BaseJavaModule;
+import com.facebook.react.devsupport.DevServerHelper;
```

Comment:
@bestandr Thanks. Trying to fix. How can I verify that buck build passes locally? Sorry for the noob question.